packages/mapviewer/src/views/EmbedView.vue

```diff
@@ -20,19 +20,67 @@ const route = useRoute()
 
 const is3DActive = computed(() => store.state.cesium.active)
 
+const { t } = useI18n()
+
+const noSimpleZoomEmbed = computed(() => store.getters.hasNoSimpleZoomEmbedEnabled)
+
+const showSimpleZoomHint = ref(false)
+let simpleZoomHintTImeout = null
+
+function onWheel(event) {
+    const isZoomModifierPressed = event.ctrlKey || event.metaKey //needed for macOS
+
+    if (noSimpleZoomEmbed.value && !isZoomModifierPressed) {
+        event.preventDefault()
+        event.stopImmediatePropagation()
+
+        showSimpleZoomHint.value = true
+        clearTimeout(simpleZoomHintTImeout)
+        simpleZoomHintTImeout = setTimeout(() => {
+            showSimpleZoomHint.value = false
+        }, 3000)
+
+        return
+    }
+}
+
```

Comment:
when the popup is displayed, ctrl+zooming when the mouse is on the popup does a browser zoom. If you can find a way to avoid that, that would be great. (an event listener on the ctrl key for example)

If it adds too much complexity, don't bother about it